babel.config.js

```diff
@@ -1,8 +1,8 @@
+const config = process.env.MOBILE ? require( './packages/react-native-editor/babel.config' ) : {
+	presets: [ '@wordpress/babel-preset-default' ],
+	plugins: [ 'babel-plugin-inline-json-import' ],
+};
 module.exports = function( api ) {
```

Comment:
Are you sure it shouldn't be using:

> `api.cache.using(() => process.env.NODE_ENV)` - Cache based on the value of `NODE_ENV`. Any time the using callback returns a value other than the one that was expected, the overall config function will be called again and a new entry will be added to the cache.

See: https://babeljs.io/docs/en/config-files#apicache

If I understand it properly, in this case, it probably would need to be:
```js
api.cache.using( () => process.env.MOBILE );
```

What I'm afraid of is that the issue with the current proposal is it will cache forever the config which is executed for the first time. It should create two cache entries, one for the mobile and one for the web.

Reply:
You're right what do you think about?

`api.cache.using( () => process.env.MOBILE ? "mobile" : "web" );`

In that case, it should create two cache entries `web` and `mobile`

Reply:
Works for me 👍